internal/engine/docker_compose_event_watcher.go

```diff
@@ -36,7 +36,7 @@ func (w *DockerComposeEventWatcher) OnChange(ctx context.Context, st store.RStor
 	configPath := state.DockerComposeConfigPath()
 	st.RUnlockState()
 
-	if configPath == "" {
+	if configPath == nil {
```

Comment:
I think this is slightly wrong. This will catch cases where the slice is nil, but not cases where the slice is empty. It's the difference between:

```golang
slice := nil
slice := []string{}
```

I think instead this should be:

```golang
if len(configPath) == 0 {
```

See this example https://play.golang.org/p/gOogOof4XI0